bedrock/mozorg/templates/mozorg/home/home.html

```diff
@@ -88,16 +88,16 @@
         <span class="toggle" role="button" aria-controls="nav-main-menu" aria-expanded="false" tabindex="0">{{_('Menu')}}</span>
         <ul id="nav-main-menu">
           <li class="first about-item">
-            <a data-link-type="about" data-element-location="nav click" href="{{ url('mozorg.about') }}">{{_('About')}}</a>
+            <a href="{{ url('mozorg.about') }}" data-link-type="nav" data-link-name="About">{{_('About')}}</a>
```

Comment:
Changing these data attributes breaks one of our functional tests. Can you please update the selectors here to match these changes? https://github.com/mozilla/bedrock/blob/master/tests/pages/base.py#L42-L44